app/src/main/java/ch/epfl/sdp/UserDataController.java

```diff
@@ -1,13 +1,10 @@
 package ch.epfl.sdp;
 
-import android.app.Activity;
-
-import java.util.List;
+import ch.epfl.sdp.db.LeaderoardViewModel;
 
 public interface UserDataController {
 
-    void loadUsersForLeaderboard(Activity activity, SetupLeaderboard setupLeaderboard, List<UserForFirebase> mUserForFirebases);
```

Comment:
Because now the data for leaderboard comes from the local caching Room(Sqlite database) and I use observer mode to notify the change of local database to UI, and I don't need this method anymore.